.github/workflows/fabric_acctest.yml

```diff
@@ -162,6 +166,20 @@ jobs:
         terraform:
           - '1.5'
     steps:
+      - id: get_id_token
+        name: Get GitHub OIDC Token for PFCR
+        uses: actions/github-script@v6
+        with:
+          script: |
+            try {
+              const idToken = await core.getIDToken('gha-fcr-client');
+              console.log('Token generated with audience: gha-fcr-client');
+              core.setOutput('id_token', idToken);
```

Comment:
Should this be using core.setSecret instead (which would influence how the value is referenced too)

https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#example-creating-an-annotation-for-an-error